### DISCUSSION TITLE
Lumus support like Neolink

### DISCUSSION THREAD

**Discussion by trapassati** [fwestenberg/reolink_dev#338] [Q&A]
Is there any possibility of integrating the Neolink code to enable Lumus support?

https://github.com/thirtythreeforty/neolink

**Comment by cpainchaud** [comment]
Hi,

 Unless I am wrong this project you are referring to is converting Reolink's proprietary video stream format to RTSP, it doesn't cover any sort of API so I would say there is little/no relationship between what we do here and they do.

Am I wrong?
